Source/WebCore/platform/audio/AudioUtilities.cpp

```diff
@@ -84,24 +84,21 @@ void applyNoise(float* values, size_t numberOfElementsToProcess, float standardD
     std::mt19937 generator(device());
     std::normal_distribution<float> distribution(1, standardDeviation);
 
-    HashMap<float, float> noiseMultipliers;
-    auto computeNoiseMultiplier = [&](float rawValue) {
-        if (!noiseMultipliers.isValidKey(rawValue))
-            return distribution(generator);
+    static constexpr auto maximumTableSize = 2048;
+    static constexpr auto minimumTableSize = 1024;
 
-        auto result = noiseMultipliers.ensure(rawValue, [&] {
-            return distribution(generator);
-        }).iterator->value;
+    size_t unusedCellCount = weakRandomNumber<uint32_t>() % (maximumTableSize - minimumTableSize);
+    size_t tableSize = maximumTableSize - unusedCellCount;
 
-        static constexpr auto maxNoiseMultiplierMapSize = 250000;
-        if (noiseMultipliers.size() >= maxNoiseMultiplierMapSize)
-            noiseMultipliers.remove(noiseMultipliers.random());
+    std::array<float, maximumTableSize> multipliers;
+    std::fill(std::begin(multipliers), std::end(multipliers), std::nanf(""));
```

Comment:
I think we can simplify this with:
```
multipliers.fill(std::numeric_limits<float>::quiet_NaN());
```

Reply:
We may also consider using `std::numeric_limits<float>::infinity()` instead NaN for the empty value, which is what our HashTable uses by default.

Reply:
Sounds good! Will go with `std::numeric_limits<float>::infinity()` and `std::isfinite`